pkgs/misc/vscode-extensions/remote-ssh/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv
+, vscode-utils
+, useLocalExtensions ? false}:
+# Note that useLocalExtensions requires that vscode-server is not running
+# on host. If it is, you'll need to remove ~/.vscode-server,
+# and redo the install by running "Connect to host" on client
+
+let
+  inherit (vscode-utils) buildVscodeMarketplaceExtension;
+  
+  # patch runs on remote machine hence use of which
+  patch = ''
+    f="/home/''$USER/.vscode-server/bin/''$COMMIT_ID/node"
+    echo PATCH: replacing ''$f with ''$(which node)
+    rm ''$f
+    ln -s ''$(which node) ''$f
```

Comment:
To solve the ubuntu problem, could this not only symlink node, if it is actually found on the remote host?